faker/providers/company/nl_NL/__init__.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+from .. import Provider as CompanyProvider
+
+
+class Provider(CompanyProvider):
+    formats = (
+        '{{last_name}} {{company_suffix}}',
+        '{{last_name}} & {{last_name}}',
+        '{{company_prefix}} {{last_name}}',
+        '{{large_company}}'

```

Comment:
we enforce trailing commas in order to make smaller diffs. You would need to add commas after the last element of any list.